### PR TITLE
Bug 571526 open license folder after issuing

### PR DESCRIPTION
**Pull request by eparovyshnaya**
Add 'open folder' to the final notification dialog

Signed-off-by: eparovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/LicenseIssuedNotification.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+				title, //
+				null, //
+				description, //
+				MessageDialog.INFORMATION, //
+				new String[] { //
+						"OK", //$NON-NLS-1$
+						"Open &Folder" //$NON-NLS-1$
+				}, 0);
+		int result = dialog.open();
+		if (result == 1) {
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/LicenseIssuedNotification.java:63]: Window.CANCEL==1 that makes this not totally clear
I suggest to use `MessageDialog.openQuestion`

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/LicenseIssuedNotification.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+			try {
+				Desktop.getDesktop().open(residence.toFile());
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/LicenseIssuedNotification.java:65]: `Program.launch`

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/LicenseIssuedNotification.java (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/LicenseIssuedNotification.java:63]: It's not a question or error reporting dialog. It is completely positive and is called only when everything is _OK_. 
A user just have an option what to do with this _OK_: let it go or have a closer look. 

I agree that customization api of MessageDialog has flaws, but the final UX of this usage would rather be kept this positive.

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/LicenseIssuedNotification.java (hunk 2 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/LicenseIssuedNotification.java]: My fault. Fixed.

**Review by ruspl-afed — APPROVED** [review]

**Pull request merged**